mlir-sycl/CMakeLists.txt

```diff
@@ -34,9 +34,12 @@ else ()
   set(MLIR_INCLUDE_DIRS ${MLIR_MAIN_SRC_DIR}/include)
   set(MLIR_CMAKE_DIR ${MLIR_MAIN_SRC_DIR}/cmake/modules)
   set(MLIR_TABLEGEN_EXE $<TARGET_FILE:mlir-tblgen>)
+  set(POLYGEIST_MAIN_SRC_DIR ${LLVM_MAIN_SRC_DIR}/../polygeist)
```

Comment:
@etiotto The buildbot doesn't build using shared libs anymore since https://github.com/intel/llvm/commit/7268336db32927413dcd2eb744bd68c42979f567 for running llvm_test_suite in buildbot.